src/Components/CreateImageWizardV2/steps/TargetEnvironment/GCP/GCPTarget.tsx

```diff
@@ -0,0 +1,218 @@
+              clearState();
+            }}
+          />
+          <Radio
+            name="gcp-share-insights-only"
+            id="gcp-share-insights-only"
+            label="Share image with Red Hat Insights only"
+            description={
+              <p>
+                Your image will be uploaded to GCP and shared with Red Hat
+                Insights.
+                <b> The image expires in 14 days.</b> You cannot access or
+                recreate this image in your GCP project.
+              </p>
+            }
+            checked={!shareGoogleAccount}
+            isChecked={!shareGoogleAccount}
+            onClick={() => {
+              setShareGoogleAccount(false);
+              clearState();
+            }}
+          />
+        </FormGroup>
+        {shareGoogleAccount && (
+          <FormGroup label="Account type" isRequired>
+            <Radio
+              name="gcp-type-google-account"
+              id="gcp-type-google-account"
+              label="Google account"
+              aria-label="Google account"
+              checked={accountType === 'googleAccount'}
+              isChecked={accountType === 'googleAccount'}
+              onClick={() => {
+                setAccountType('googleAccount');
+                clearValues();
+              }}
+            />
+            <Radio
+              name="gcp-type-service-account"
+              id="gcp-type-service-account"
+              label="Service account"
+              checked={accountType === 'serviceAccount'}
+              isChecked={accountType === 'serviceAccount'}
+              onClick={() => {
+                setAccountType('serviceAccount');
+                clearValues();
+              }}
+            />
+            <Radio
+              name="gcp-type-google-group"
+              id="gcp-type-google-group"
+              label="Google group"
+              checked={accountType === 'googleGroup'}
+              isChecked={accountType === 'googleGroup'}
+              onClick={() => {
+                setAccountType('googleGroup');
+                clearValues();
+              }}
+            />
+            <Radio
+              name="gcp-type-domaing"
+              id="gcp-type-domaing"
+              label="Google Workspace domain or Cloud Identity domain"
+              checked={accountType === 'domain'}
+              isChecked={accountType === 'domain'}
+              onClick={() => {
+                setAccountType('domain');
+                clearValues();
+              }}
+            />
+          </FormGroup>
+        )}
+        {shareGoogleAccount && accountType !== 'domain' && (
+          <ValidatedTextField
+            aria="GCP account email"
+            label="Principal (e.g e-mail address)"
+            fieldId="gcp-account-email"
+            value={accountEmail}
+            setValue={setAccountEmail}
+            validateFunction={validateEmail}
+            helperText="must be a valid email address"
```

Comment:
The label of the text field says "e-mail", let's keep it the same in the error.

```suggestion
            helperText="Must be a valid e-mail address"
```

Also similar question as for the AWS validated fields - would it be possible to show the error when there's something wrong going on instead of from the very beginning when the field was not even interacted with?